model/mcp/registry.yaml

```diff
@@ -0,0 +1,170 @@
+              brief: >
+                Notification indicating that a resource has been updated.
+              stability: development
+            - id: prompts_list
+              value: prompts/list
+              brief: >
+                Request to list prompts available on server.
+              stability: development
+            - id: prompts_get
+              value: prompts/get
+              brief: >
+                Request to get a prompt.
+              stability: development
+            - id: notifications_prompts_list_changed
+              value: notifications/prompts/list_changed
+              brief: >
+                Notification indicating that the list of prompts has changed.
+              stability: development
+            - id: tools_list
+              value: tools/list
+              brief: >
+                Request to list tools available on server.
+              stability: development
+            - id: tools_call
+              value: tools/call
+              brief: >
+                Request to call a tool.
+              stability: development
+            - id: notifications_tools_list_changed
+              value: notifications/tools/list_changed
+              brief: >
+                Notification indicating that the list of tools has changed.
+              stability: development
+            - id: logging_set_level
+              value: logging/setLevel
+              brief: >
+                Request to set the logging level.
+              stability: development
+            - id: notifications_message
+              value: notifications/message
+              brief: >
+                Notification indicating that a message has been received.
+              stability: development
+            - id: sampling_create_message
+              value: sampling/createMessage
+              brief: >
+                Request to create a sampling message.
+              stability: development
+            - id: completion_complete
+              value: completion/complete
+              brief: >
+                Request to complete a prompt.
+              stability: development
+            - id: roots_list
+              value: roots/list
+              brief: >
+                Request to list roots available on server.
+              stability: development
+            - id: notifications_roots_list_changed
+              value: notifications/roots/list_changed
+              brief: >
+                Notification indicating that the list of roots has changed.
+              stability: development
+      - id: mcp.request.id
+        type: string
+        brief: >
+          This is a unique identifier for the request.
+        stability: development
+        examples: ["42"]
+      - id: mcp.session.id
+        type: string
+        brief: Identifies MCP session.
+        stability: development
+        examples: ["191c4850af6c49e08843a3f6c80e5046"]
+      - id: mcp.tool.name
```

Comment:
Maybe a simliar issue as https://github.com/open-telemetry/semantic-conventions/issues/2112, usually for a mcp server, it will contain multiple tools, how the semantic convention cover this case?

Reply:
this is for a request/ specific [tool call](https://github.com/modelcontextprotocol/modelcontextprotocol/blob/44029de47cfcf98f6214e40cbdce1e412612c088/schema/2025-03-26/schema.ts#L715C1-L715C18) - there is just one, I don't cover the list of tool definitions here (or batching). 

Once we do this, we'll probably need to record tool definitions as an array and/or tool names as array.

Recording a single tool name on a specific tool call is important, I don't think we should convert it to a list just for the sake of consistency.